### ISSUE TITLE
Issues installing dependencies with ONLY "^" value

### ISSUE THREAD

**Issue by chungusg** [saicaca/fuwari#16]
I had issues with using "npm install" to install dependencies, it would return **npm ERR! Invalid tag name "^" of package "@astrojs/check@^":** I eventually fixed this by adding the most recent version after the "^" (e.g."^5.3.3") for both @astrojs/check and typescript. I was wondering if this was an issue only I had, or a re-occurring problem?

**Comment by saicaca** [comment]
That's unusual. The version number does exist in [package.json](https://github.com/saicaca/fuwari/blob/main/package.json). `npm install` has been working fine on my end and I have no idea why the version number would be missing. Do you have a method to reproduce the error?

**Comment by chungusg** [comment]
You're right! Somehow I didn't think to check the original file here in your repository, but you're right the version number IS there.  Although WHY my package.json file is different is a mystery.

I did a clean install of the latest version of Nodejs and imported this template through the npm create astro@latest -- --template "REPO" command, and I did this multiple times and on two different computers. I ended up with the same errors while trying to install dependencies in the vscode (version 1.86.2) terminal.

I also tried importing other astro templates, and I didn't come across this issue.

It seems to only be an issue in my case, so I'm glad!
